polling_stations/apps/data_importers/management/commands/import_south_staffs.py

```diff
@@ -2,87 +2,52 @@
 
 
 class Command(BaseXpressDemocracyClubCsvImporter):
-    council_id = "E07000196"
+    council_id = "SST"
     addresses_name = (
-        "parl.2019-12-12/Version 1/Democracy_Club__12December2019sstaffs.tsv"
+        "2021-03-23T12:44:29.340168/South Staffs Democracy_Club__06May2021.CSV"
     )
     stations_name = (
-        "parl.2019-12-12/Version 1/Democracy_Club__12December2019sstaffs.tsv"
+        "2021-03-23T12:44:29.340168/South Staffs Democracy_Club__06May2021.CSV"
     )
-    elections = ["parl.2019-12-12"]
-    csv_delimiter = "\t"
-    allow_station_point_from_postcode = False
-
-    def station_record_to_dict(self, record):
-
-        if record.polling_place_id == "3464":
-            record = record._replace(
-                polling_place_easting="391056", polling_place_northing="319891"
-            )
```

Comment:
Can you carry this fix over please. PS name is: `Village Hall, Hyde Lea` and new id is `3849`